calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv

```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1466,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1466_Calibration_2019-06-17.zip > BBFL2-1466 Charsheet.pdf
```

Comment:
I think the source_file note should read
FLORT-K_BBFL2_SN_1466_Calibration_2020-07-15 > BBFL2-1466 Charsheets.pdf

Reply:
updated to new vendor cal doc and cal coefficients.